pkgs/applications/video/gyroflow/default.nix

```diff
@@ -73,8 +88,8 @@ rustPlatform.buildRustPackage rec {
 
   # These variables are needed by gyroflow/build.rs.
   # OPENCV_LINK_LIBS is based on the value in gyroflow/_scripts/common.just, with opencv_dnn added to fix linking.
-  env.OPENCV_LINK_PATHS = "${opencv}/lib";
-  env.OPENCV_LINK_LIBS = "opencv_core,opencv_calib3d,opencv_dnn,opencv_features2d,opencv_imgproc,opencv_video,opencv_flann,opencv_imgcodecs,opencv_objdetect,opencv_stitching,png";
+  env.OPENCV_LINK_PATHS = "${opencv4}/lib";
+  env.OPENCV_LINK_LIBS = "opencv_core,opencv_calib3d,opencv_features2d,opencv_imgproc,opencv_video,opencv_flann,opencv_dnn";
```

Comment:
What is the reason for removing opencv_imgcodecs et al.? gyroflow build script still links to them: https://github.com/gyroflow/gyroflow/blob/v1.6.0/_scripts/common.just#L38

Reply:
I do not suggest setting `env.OPENCV_LINK_LIBS` to `"opencv_core4,...` (the value in `common.just`) - opencv does not install libs suffixed by `4`, and the link fails. I suggest leaving `env.OPENCV_LINK_LIBS` the way it was before this PR (unless there is a reason to change it). This PR builds fine for me with the old value.

Reply:
Currently `default.nix` has `env.OPENCV_LINK_LIBS = "opencv_core,opencv_calib3d,opencv_dnn,opencv_features2d,opencv_imgproc,opencv_video,opencv_flann,opencv_imgcodecs,opencv_objdetect,opencv_stitching,png";` and the comment above says that it is based on `gyroflow/_scripts/common.just` which links to all these libraries.
You change it to `env.OPENCV_LINK_LIBS = "opencv_core,opencv_calib3d,opencv_features2d,opencv_imgproc,opencv_video,opencv_flann,opencv_dnn";`. Could you either explain this change in the comment above if there is a reason to do it, or discard this change and keep this line unmodified?

Reply:
Please do not change this